docs/source/how_to_guides/inference/how_to_do_bootstrap_inference.ipynb

```diff
@@ -0,0 +1,1038 @@
+   "outputs": [],
+   "source": [
+    "def ols_fit(data):\n",
+    "\n",
+    "    y = data[\"pulse\"]\n",
+    "    x = data[[\"constant\", \"time\"]]\n",
+    "    params = sm.OLS(y, x).fit().params\n",
+    "    return params"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The outcome function must return a pandas Series.\n",
+    "\n",
+    "Now we wan calculated confidence intervals and standard errors."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>mean</th>\n",
+       "      <th>std</th>\n",
+       "      <th>lower_ci</th>\n",
+       "      <th>upper_ci</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>constant</th>\n",
+       "      <td>93.744999</td>\n",
+       "      <td>1.517731</td>\n",
+       "      <td>90.876828</td>\n",
+       "      <td>96.816027</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>time</th>\n",
+       "      <td>0.388968</td>\n",
+       "      <td>0.123150</td>\n",
+       "      <td>0.148330</td>\n",
+       "      <td>0.639725</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "               mean       std   lower_ci   upper_ci\n",
+       "constant  93.744999  1.517731  90.876828  96.816027\n",
+       "time       0.388968  0.123150   0.148330   0.639725"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from estimagic.inference import bootstrap\n",
+    "\n",
+    "results_without_cluster = bootstrap(data=df, outcome=ols_fit)\n",
+    "results_without_cluster"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The above function call represents the minimum that a user has to specify, making full use of the default options, such as drawing a 1000 bootstrap draws, using the \"percentile\" bootstrap confidence interval, not making use of parallelization, etc.\n",
+    "\n",
+    "If we would for example want to make 10000 draws, while parallelizing the maximum possible number of threads and using a \"bca\" type confidence interval, we would simply call the following."
```

Comment:
Is ``n_cores=1`` inducing the usage of the "maximum possible number of threads"?

Reply:
Good catch. I changed it from `-1` to `1`. Now changing it to `2` to show how parallelization works. Don't want to encourage `-1` because it is typically too much parallelization (at least on processors with hyperthreading).